src/Components/InputWithLabel.js

```diff
@@ -30,4 +31,13 @@ function InputWithLabel({
   )
 }
 
+InputWithLabel.propTypes = {
+  id: PropTypes.string.isRequired,
+  todoTitle: PropTypes.string.isRequired,
+  type: PropTypes.string,
+  handleTitleChange: PropTypes.func.isRequired,
+  children: PropTypes.string,
+  isFocused: PropTypes.bool.isRequired,
```

Comment:
Everything looks good Vini!!! 💯 

Only nitpik here is maybe leaving isFocus not required since this is a generic component. In our app, its required but in another, it may not be. Something to keep in mind if you decide to use code from old projects